packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/observe/TransactionTableRow.tsx

```diff
@@ -97,39 +98,43 @@ export function TransactionTableRow({ row }: { row: Row<WatchedTransaction> }) {
                   </>
                 ) : null}
 
-                <Separator className="my-5" />
+                {!data.error ? (
+                  <>
+                    <Separator className="my-5" />
```

Comment:
do you know about tailwind's `divide` classes? https://tailwindcss.com/docs/divide-width

Reply:
oh interesting, wasn't aware of it. I'll explore it more when working on the layout